app/models/lookup_value.rb

```diff
@@ -23,6 +23,14 @@ class LookupValue < ActiveRecord::Base
   scoped_search :on => :match, :complete_value => true
   scoped_search :in => :lookup_key, :on => :key, :rename => :lookup_key, :complete_value => true
 
+  def value=(val)
+    if val.is_a?(Hash)
```

Comment:
``if val.is_a?(HashWithIndifferenAccess)`` might be a bit more self-explanatory (my question few months later would be: why do we need to call to_hash on Hash…